### PR TITLE
Added Droidcon PL

### PR DESCRIPTION
**Pull request by a11n**

### CODE REVIEW

File: _conferences/droidcon-pl-2015.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: "Droidcon"
+website: http://droidcon.pl
+location: Kraków, PL
+
+date_start: 2015-12-04
+date_end:   2015-12-06
+
+cfp_site: http://goo.gl/forms/mfJxBfi7L3
```
Comment (JakeWharton) [inline _conferences/droidcon-pl-2015.md:11]: This won't show up unless there's `cfp_start` and `cfp_end` dates.

Comment (a11n) [inline _conferences/droidcon-pl-2015.md:11]: Unfortunately they haven't specified a CFP end date :-(
Should I assume a date, for instance start date of conference or delete cfp_site?

Comment (JakeWharton) [inline _conferences/droidcon-pl-2015.md:11]: Just put it as starting yesterday and ending 4 weeks before the conference. If the end date changes, we can update the site.

Comment (a11n) [inline _conferences/droidcon-pl-2015.md:11]: Sorry for inconvenience.

I updated the PR as suggested.

**Comment by JakeWharton** [comment]
Thanks!

**Pull request merged**